genericjpa/async-updates/src/main/java/org/hibernate/search/genericjpa/jpa/util/impl/JPAQueryWrapper.java

```diff
@@ -0,0 +1,43 @@
+package org.hibernate.search.genericjpa.jpa.util.impl;
+
+import javax.persistence.Query;
+import java.util.List;
+
+import org.hibernate.search.genericjpa.jpa.util.impl.QueryWrapper;
+
+/**
+ * Created by Martin on 11.11.2015.
```

Comment:
yeah, seems like you're right, but these were automatically generated (and i haven't changed the default).